cmd/unbind/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/miekg/dns"
+	"log"
+	"net"
+	"regexp"
+	"strconv"
+)
+
+
+func domainsToAddresses (domain string) (string, bool) {
+	ok := false
+	ip := ""
+	re := regexp.MustCompile(`(ec2|ip)-(\d{1,3})-(\d{1,3})-(\d{1,3})-(\d{1,3})\.(compute|ec2).*`)
```

Comment:
We need to support arbitrary suffixes here. In fact, I would prefer to make this mapping configurable. That is, we should allow any number of mappings from regex string to result (with backreference support).